libminifi/src/ThreadedSchedulingAgent.cpp

```diff
@@ -64,11 +68,18 @@ void ThreadedSchedulingAgent::schedule(Processor *processor)
 		return;
 	}
 
+	auto processContext = std::make_shared<ProcessContext>(processor);
```

Comment:
Why are you getting the raw (stored -- not managed ) pointer if you're making a shared ptr?